Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM registry.svc.ci.openshift.org/openshift/release:golang-1.10 AS builder
+FROM registry.svc.ci.openshift.org/openshift/release:golang-1.11 AS builder
```

Comment:
Should be golang 1.12.

Also be sure to have a follow-up PR to update the prow config in openshift/release

Reply:
Updated & created